Source/WTF/Scripts/generate-platform-args

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python3
+import subprocess
+import os
+import shlex
+import re
+
+framework_search_paths = shlex.split('{BUILT_PRODUCTS_DIR} {FRAMEWORK_SEARCH_PATHS} '
+                                     '{SYSTEM_FRAMEWORK_SEARCH_PATHS}'.format_map(os.environ))
+header_search_paths = shlex.split('{BUILT_PRODUCTS_DIR} {HEADER_SEARCH_PATHS} '
+                                  '{SYSTEM_HEADER_SEARCH_PATHS}'.format_map(os.environ))
+preprocessor_definitions = shlex.split('RELEASE_WITHOUT_OPTIMIZATIONS '
+                                       '{GCC_PREPROCESSOR_DEFINITIONS}'.format_map(os.environ))
+
+depfile = '{DERIVED_FILES_DIR}/{INPUT_FILE_BASE}.{arch}.d'.format_map(os.environ)
+
+target_triple = '{arch}-{LLVM_TARGET_TRIPLE_VENDOR}-{LLVM_TARGET_TRIPLE_OS_VERSION}'.format_map(os.environ)
+target_triple += os.environ.get('LLVM_TARGET_TRIPLE_SUFFIX', '')
+
+swift_args = open(os.environ['SCRIPT_OUTPUT_FILE_0'], mode='w')
+
+feature_and_platform_defines = subprocess.check_output(
+    ('clang', '-std=c++2b', '-x', 'c++', '-target', target_triple,
```

Comment:
Will it automatically know to use the correct toolchain, or do you need `xcrun` here?

Also, it's unfortunate to add another place where `-std` is defined, as this increases the risk of getting out of sync.